distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -91,18 +98,16 @@ new_builder (capacity=10) = Builder.new capacity
    A vector allows to store an arbitrary number of elements in linear memory. It
    is the recommended data structure for most applications.
 from_polyglot_array : Any -> Vector Any
-from_polyglot_array arr = Vector_Data (Proxy_Polyglot_Array.Proxy_Polyglot_Array_Data arr)
+from_polyglot_array array = @Builtin_Method "Vector.from_polyglot_array"
 
 ## The basic, immutable, vector type.
+@Builtin_Type
 type Vector a
 
     ## ADVANCED
```

Comment:
Need to clean up the left over doc comment (probably attach to the type now).

Reply:
I have just modified the documentation in [a34e734](https://github.com/enso-org/enso/pull/3667/commits/a34e734d7aff6b1a5c94e5931d2a9251dd04253c). I hope it is correct, but I don't know how to generate HTML from the doc to verify... anyway resolving.